service/controller/app/v1/resource/chartoperator/resource.go

```diff
@@ -135,41 +134,35 @@ func (r Resource) installChartOperator(ctx context.Context, cr v1alpha1.App) err
 	}
 
 	{
-		err = cc.Clients.Helm.InstallReleaseFromTarball(ctx, tarballPath, namespace, helm.ReleaseName(release), helm.ValueOverrides(chartOperatorValues))
-		if err != nil {
-			return microerror.Mask(err)
-		}
-	}
-
-	{
-		// We wait for the chart-operator deployment to be ready so the
-		// chart CRD is installed. This allows the chart
-		// resource to create CRs in the same reconcilation loop.
-		r.logger.LogCtx(ctx, "level", "debug", "message", "waiting for ready chart-operator deployment")
-
-		o := func() error {
-			err := r.checkDeploymentReady(ctx, cc.Clients.K8s.K8sClient())
-			if err != nil {
-				return microerror.Mask(err)
-			}
+		r.logger.LogCtx(ctx, "level", "error", "message", "creating chart-operator release")
+
+		ch := make(chan error)
+
+		// We update the helm release but with a short timeout so we don't
+		// block reconciling other CRs. This gives time to make the port
+		// forwarding connection to the Tiller API.
+		//
+		// If we do timeout the install will continue in the background.
+		// We will check the progress in the next reconciliation loop.
+		go func() {
+			err = cc.Clients.Helm.InstallReleaseFromTarball(ctx, tarballPath, namespace, helm.ReleaseName(release), helm.ValueOverrides(chartOperatorValues))
+			close(ch)
+		}()
```

Comment:
Uses the same approach as chart-operator with a short timeout. So we don't block.